.github/workflows/build.yml

```
@@ -0,0 +1,89 @@
+          docker run --rm -v $(pwd):/data ghcr.io/erebe/wstunnel:latest
+          ./wstunnel
+            
+      - uses: actions/upload-artifact@v1
+        with:
+          name: wstunnel-linux-x64
+          path: wstunnel
+          
+  build-windows:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)